guides/common/modules/proc_synchronizing-python-repositories.adoc

```diff
@@ -21,6 +21,8 @@ Clear this field if the repository does not require authentication.
 Clear this field if the repository does not require authentication.
 . Optional: In the *Excludes* field, enter a list of Python packages separated by newlines to exclude from synchronizing to {Project}.
 . Optional: In the *Includes* field, enter a list of Python packages separated by newlines to limit the synchronization only to the included packages.
+. Optional: In the *Keep latest packages* field, enter a number to limit the amount of Python packages kept after synchronization.
+By default, {Project} keeps all package versions.
```

Comment:
I'm wondering what the number means. How does it select the packages to keep? If I enter 2, will it keep two latest versions of each Python package?
```suggestion
. Optional: In the *Keep latest packages* field, enter a number to limit the amount of Python package versions kept after synchronization.
By default, {Project} keeps all package versions.
```

Reply:
Yes, it will keep the latest N versions per package. Applied your suggestion.